src/cltkv1/utils/data_types.py

```diff
@@ -78,6 +78,15 @@ class Doc:
     Contains overall attributes of submitted texts, plus most
     importantly the processed tokenized text ``words``,
     being a list of ``Word`` types.
+
+    >>> from cltkv1 import NLP
+    >>> from cltkv1.utils.example_texts import get_example_text
+    >>> cltk_nlp = NLP(language="lat")
+    >>> cltk_doc = cltk_nlp.analyze(text=get_example_text("lat"))
+    >>> cltk_doc.raw[:38]
+    'Gallia est omnis divisa in partes tres'
+    >>> isinstance(cltk_doc.raw, str)
+    True
     """
 
     indices_sentences: List[List[int]] = None
```

Comment:
I think we should get rid of `indices_sentences` and `indices_tokens` and use @propery instead. Sounds right?

Reply:
I didn't find a use for `indices_sentences`.  We need `indices_tokens` to be able to reconstruct the sentences though?

Reply:
> to reconstruct the sentences

Couldn't we just embed this info into the `List[Word]` of `Doc.words`. That way, all our persisted data is contained within that one place.

Also, I remember hearing that an upcoming release of stanfordnlp will have word boundary indices.

Reply:
Yes, sure.  The term `indices` put me in a database frame of mind, where you have the list of your data, and then various indices to pick things out.  But given that the actual NLP work is so much slower than walking through the `words` building sentences, the cost of just keeping everything in one place seems minor.